linkerd/core/src/main/scala/io/buoyant/linkerd/SvcConfig.scala

```diff
@@ -33,9 +33,11 @@ trait SvcConfig {
    * responseClassifier categorizes responses to determine whether
    * they are failures and if they are retryable.
    */
-  @JsonProperty("responseClassifier")
   var _responseClassifier: Option[ResponseClassifierConfig] = None
 
+  @JsonProperty("responseClassifier")
+  def responseClassifier_=(r: ResponseClassifierConfig): Unit = _responseClassifier = Some(r)
```

Comment:
I think `getResponseClassifier` is fine.  We're outside the idiom because of this little hack.  As long as it has the right JsonProperty annotation so that it serializes correctly.